### ISSUE TITLE
User account transfer/access in the event of death

### ISSUE THREAD

**Issue by FluxedScript** [supertiger1234/Nertivia-Client#252]
Let's say you have no real-life friends and no family. You can't give your google account with all those dodgy things on there or address to online friends. What if you had a setting for after `n` days/months inactive you can give your account access to someone else. Helpful if you run a large server as you wouldn't want to give people access if you were still using it.

**Comment by Toby222** [comment]
Or just have your online data get deleted after some time of inactivity?

**Comment by FluxedScript** [comment]
Not if you want your friend to take over when you die to finish some questions

**Comment by geeklystudios** [comment]
honestly a good idea, but what if you are just taking a break from it?

**Comment by FluxedScript** [comment]
@geeklystudios then you would define how long you were taking a break then the timer starts again

**Comment by skspuddd** [comment]
So this is kind of like a vacation timer?

**Comment by FluxedScript** [comment]
@skspuddd let's say you were to set me as your backup account, if you go on Holiday one day in Brazil and never come back maybe you found a new life or something bad happened. In this example, you have hundreds of people who need you for a project as you are developing it. So you set it for 3 months, maybe you went away from social media and the online world for a while then you can just set it for a longer period of time. 

Those hundreds of people need information on why you aren't coming back. Maybe you lost access to your account one day because your phone, laptop, and all other internet-accessible devices got burned in a fire. Now you can't use 2fa as you lost your backup codes and authenticator app. Yes, there are online versions of this but let's say you lost everything. You remember you have an alt account, you could go saying that you were in an accident, but then no one would believe you.

This feature basically aims to help transfer your account to another basically like managed access since they don't need your password or 2fa code but they need to wait for it. Not many people trust their families with their passwords and especially with my habits I wouldn't trust mine either. I do put more trust in people on the internet who I have known for a long time but still don't want to send my password to because you never know what they will do with it. 

You could also use this in a case with a large influencer account. If I use this platform to communicate with millions of people, they want to send my account to my child(If I ever have one) if I don't get on within a certain amount of days. You wouldn't want to give them full access since they are still slightly immature and could just start ranting on about this Minecraft server owner who banned them. 

Also the opportunity to have multiple accounts if they weren't able to respond let's say you had a designated survivor in your friendship group who sits at home away from danger. Maybe you don't like them as much as your other friends so you set them to a 5th position. Now if you and those 4 people were to go on holiday and the plane were to have a crash or a natural disaster happened you wouldn't lose complete access to that account. 

In short: no, it's more of the last step to help continue your account.

**Comment by FluxedScript** [comment]
I love the essay I wrote
